docs/current_docs/api/snippets/fs-filters/pre-call/java/MyModule.java

```diff
@@ -0,0 +1,23 @@
+package io.dagger.modules.mymodule;
+
+import io.dagger.module.annotation.Default;
+import io.dagger.module.annotation.Function;
+import io.dagger.module.annotation.Object;
+
+@Object
+public class MyModule {
+  private String greeting;
+  private String name;
+
+  public MyModule() {}
+
+  public MyModule(@Default("Hello") String greeting, @Default("World") String name) {
+    this.greeting = greeting;
+    this.name = name;
+  }
+
+  @Function
+  public String message() {
+    return greeting + " " + name;
+  }
+}
```

Comment:
This is not the right snippet, it should filters files instead of being the default hello world

Reply:
Fixed 👍

Reply:
Did you push your changes? I still see the wrong snippet :/

Reply:
🙄 I'm pretty sure I had, but it looks like not... 🤔 
I'll fix that again

Reply:
Should be good this time